upgrade/helpers/tools.py

```diff
@@ -127,6 +127,26 @@ def host_ssh_availability_check(host, timeout=7):
             time.sleep(5)
 
 
+def disable_old_repos(repo_name, timeout=1):
+    """This ensures that the repo is disable and the command doesn't timeout
+
+    :param repo_name: Repo ID of the repo to be disable
+    :param int timeout: The polling timeout in minutes.
```

Comment:
why not use seconds straight away? it's elementary school maths :)

Reply:
It would be good to increase/decrease timeout value in future, and in almost all places we are using minutes in upgrade.

Reply:
In general, we are using minutes format every where in upgrade automation. and in any case its not a blocking comment for this PR.